src/Components/Patient/FileUpload.tsx

```diff
@@ -905,11 +905,11 @@ export const FileUpload = (props: FileUploadProps) => {
   }
 
   const onFileChange = (e: ChangeEvent<HTMLInputElement>): any => {
-    if (e.target.files == null) {
-      throw new Error("Error finding e.target.files");
+    if (e.target.files == null || e.target.files.length === 0) {
+      return;
     }
     const f = e.target.files[0];
-    const fileName = f.name;
+    const fileName = f ? f.name : Date.now().toString(); // use timestamp as filename for unnamed files
```

Comment:
Why was this changed? `f` would never be evaluated to be false since we are doing an early exit in line 909, and hence the Date.now() part would never be used.